### ISSUE TITLE
border colour ratio with body-background is too low

### ISSUE THREAD

**Issue by tombye** [alphagov/govuk-frontend#6776]
## Description of the issue

The [border colour](https://design-system.service.gov.uk/styles/colour/#:~:text=border,-#cecece) (#cecece) has a contrast ratio of 2:1:1 with the [body-background](https://design-system.service.gov.uk/styles/colour/#:~:text=body-background) (#ffffff) colour. This was raised as lower than the minimum of 3:1 required by the Non-text contrast (AA) WCAG success criterion in an accessibility audit done on the GOV.UK Notify service in September 2024.

It's worth noting that the auditor raised this as a usability issue, not a WCAG fail. They raised it against Notify's use of the summary list component but I've framed this around the 2 colours mentioned because it will apply to other components/patterns.

The audit can be viewed here, with the issue on page 49:

https://drive.google.com/file/d/1qR8t3mcEka6sUVGqfhnjEpyhqqQhGNFi/view?usp=sharing

## Steps to reproduce the issue

Use a colour contrast checker like [webaim's online checker](https://webaim.org/resources/contrastchecker/) to check the 2 colours.

## Actual vs expected behaviour

The border colour should have a ratio of 3:1 with the body-background colour.

## Environment (where applicable)

- Operating system: All OS's but checked on MacOS
- Browser: Google Chrome
- Browser version: 145.0.7632.110 (Official Build) (arm64)
- GOV.UK Frontend Version: v6.0.0

**Comment by querkmachine** [comment]
@tombye The 3:1 minimum non-text contrast criterion applies to user interface components, which WCAG defines as "a part of the content that is perceived by users as a single control for a distinct function".

I've had a quick dig around and it doesn't look like we use the border colour as the sole means of identifying any interactive controls. It's overwhelmingly used as a border on non-interactive elements, and often in addition to other means of identification or separation, like alignment or spacing differences. 

The uses I found in components: 

* **Accordion:** Separator between sections/interactive controls.
* **Checkboxes** and **radio buttons**: Border on non-interactive, conditionally revealed content. 
* **Details**: Border on non-interactive, conditionally revealed content.
* **Inset text:** Border on non-interactive content.
* **Pagination:** Separator between links on the block variant. 
* **Phase banner:** Border on non-interactive content.
* **Summary list:** Separator between action links, between content rows, and around card groups. 
* **Table:** Separator between content rows.
* **Tabs:** Border around active tabs (these are technically interactive, but already in their active state, so might be a bit of a grey area) and around content panels. 
* **Task list:** Separator between content rows.

The colour is also used for section breaks (horizontal rules), which are non-interactive. 

We could do more here, such as darkening the border based on the user's `prefers-contrast` preference, but as it stands, I don't think this is a WCAG issue nor a bug in the typical sense, so I'm gonna switch out the tags to mark it as a possible enhancement instead.
